changelog.md

```diff
@@ -16,6 +16,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   will from now on be made available through the system tray application and you
   will have the choice to install it now or later.
 
+### Fixed
+
+- Fixed logic checking for deprecations.
```

Comment:
Remember to phrase this with end users in mind. What does this fix mean practically for end-users?